### PR TITLE
Players will no longer be shoved around by weak wind

### PR DESCRIPTION
**Pull request by MaxIsJoe**
Title
Especially useful in lavaland where you can get interrupted by wind when mining a single tile.

CL: [Fix] - Players will no longer be shoved around by weak wind

### CODE REVIEW

**Comment by corp-0** [comment]
looks good to me but I will wait for Bod since he knows better the movement code

File: UnityProject/Assets/Scripts/Player/MovementV2/MovementSynchronisation.cs [hunk 1]
```diff
@@ -372,7 +380,7 @@ public void CheckWindOtherPush()
 
 			if (Data.magnitude > 0.1f)
 			{
-				if (PushingData != Data)
+				if (PushingData != Data && Data.magnitude > (int)WindStrength.WEAK)
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Player/MovementV2/MovementSynchronisation.cs]: 8 tiles per second is pretty damn fast, that's the same as player movement pretty much

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Player/MovementV2/MovementSynchronisation.cs]: > 8 tiles per second is pretty damn fast, that's the same as player movement pretty much

All wind push movements I've tested have been very high.
These are numbers i've debugged, the server pushes the player 8 tiles when mining anything and the space vacuum is always 12 or above.

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Player/MovementV2/MovementSynchronisation.cs]: I'm just saying I've changed how space wind works in the other PR

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Player/MovementV2/MovementSynchronisation.cs]: then merge your PR so i can make my changes or merge this first and change it on your end if the review process is going to be long

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Player/MovementV2/MovementSynchronisation.cs]: Someone needs to Approve it though

File: UnityProject/Assets/Scripts/Player/MovementV2/MovementSynchronisation.cs [hunk 2]
```diff
@@ -331,16 +329,6 @@ public void Update()
 			ServerCheckQueueingAndMove();
 		}
 		
-		if (Intangible == false && CanBeWindPushed)
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Player/MovementV2/MovementSynchronisation.cs:334]: Looks like a bunch of accidental changes?

**Comment by corp-0** [comment]
what's up with this one?

**Comment by MaxIsJoe** [comment]
git shinangins 
if anyone wants to claim this PR and implament the wind push limit thing then be my guest because it's actually just a few lines.

**Pull request closed without merging**